frameit/lib/frameit/editor.rb

```diff
@@ -238,12 +245,13 @@ def put_title_into_background(background)
     end
 
     def actual_font_size
-      [@image.width / 10.0].max.round
+      font_scale_factor = fetch_config['font_scale_factor'] || 0.1
+      [@image.width * font_scale_factor].max.round
     end
 
     # The space between the keyword and the title
     def keyword_padding
-      (actual_font_size / 2.0).round
+      (actual_font_size / 3.0).round
```

Comment:
I'm a bit worried about changing this out from under existing `frameit` users. What if we kept this value as 2.0 by default and added a `keyword_padding_factor` configuration item to allow custom values to be specified?